README.md

```diff
@@ -68,6 +68,9 @@ Attributes
 * `ntp['peer']['maxpoll']` (applies to NTP Servers ONLY)
   - Boolean. Defaults to 10 (ntp default). Specify the maximum poll intervals for NTP messages, in seconds to the power of two.
 
+* `ntp['server']['prefer']` (applies to NTP Servers and Clients)
+  - String. No Default. The server from `ntp['servers']` to prefer getting the time from.
```

Comment:
> No Default

Should this be `Defaults to an empty string` or something similar? `No default` to me means the default is `nil`.  Thoughts?